docs/literate/src/files/first_steps/create_first_setup.jl

```diff
@@ -0,0 +1,258 @@
+#src # Create first setup
+
+# In this part of the introductory guide, we will create a first Trixi.jl setup as an extension of
+# [`elixir_advection_basic.jl`](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/tree_2d_dgsem/elixir_advection_basic.jl).
+# Since Trixi.jl has a common basic structure for the setups, you can create your own by extending
+# and modifying the following example.
+
+# Let's consider the linear advection equation for a state ``u = u(x, y, t)`` on the two-dimensional spatial domain
+# ``[-1, 1] \times [-1, 1]`` with a source term
+# ```math
+# \frac{\partial}{\partial t}u + \frac{\partial}{\partial x} (0.2 u) - \frac{\partial}{\partial y} (0.7 u) = - 2 e^{-t}
+# \sin\bigl(2 \pi (x - t) \bigr) \sin\bigl(2 \pi (y - t) \bigr)
+# ```
+# and with an initial condition
+# ```math
+# u(x, y, 0) = \sin\bigl(\pi x \bigr) \sin\bigl(\pi y \bigr).
+# ```
+
+# The first step is to create and open a file with the .jl extension. You can do this with your
+# favorite text editor. In this file you will create your setup.
```

Comment:
Maybe mention that VSCode is the recommended choice ?

Reply:
```suggestion
# favorite text editor (if you do not have one, we recommend [VS Code](https://code.visualstudio.com/)).
# In this file you will create your setup.
```